core/templates/dev/head/components/summary_tile/CollectionSummaryTileDirective.js

```diff
@@ -34,17 +34,20 @@ oppia.directive('collectionSummaryTile', [
         getThumbnailIconUrl: '&thumbnailIconUrl',
         getThumbnailBgColor: '&thumbnailBgColor',
         isLinkedToEditorPage: '=?isLinkedToEditorPage',
-        getCategory: '&category'
+        getCategory: '&category',
+        isPlaylistMode: '&playlistMode',
```

Comment:
isInPlaylistMode reads better, I think. Or isPlaylistTile. (But this tile isn't a "mode".)

Reply:
Done!